modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionOverview/components/SubscriptionsNavbar/SubscriptionNavbar.js

```diff
@@ -12,9 +12,9 @@
 import {Button as ClayButton, DropDown} from '@clayui/core';
 import ClayIcon from '@clayui/icon';
 import React, {useEffect, useRef, useState} from 'react';
-import i18n from '../../../../common/I18n';
-import RoundedGroupButtons from '../../../../common/components/RoundedGroupButtons';
-import {useCustomerPortal} from '../../context';
+import i18n from '../../../../../../../../../common/I18n';
+import RoundedGroupButtons from '../../../../../../../../../common/components/RoundedGroupButtons';
+import useUpdateShowDropDown from './hooks/useUpdateShowDropDown';
 
 const SubscriptionDropDownMenu = ({
```

Comment:
I believe we should move the Subscription DropDown for another component outside the navbar because the navbar is one component of the subscriptionsOverview and the dropdown is another one. Also since is more than one subscription, we should name it SubscriptionsDropDownMenu